### ISSUE TITLE
Can´t make it work. Camera is flashed but it will not connect. Log inside

### ISSUE THREAD

**Issue by egidoval** [EliasKotlyar/Xiaomi-Dafang-Hacks#453]
Hello everyone.
I flashed the camera and it seems i did it right. Problem is when modifying the "Supplicant conf" 
I turn on the camera and this is the log i´m getting.
==================================================
Starting the Dafang Hacks Custom Application Layer
==================================================
Loaded common functions
Created etc directory on sdcard
Bind mounted /system/sdcard/root to /root
Bind mounted /system/sdcard/etc to /etc
Created cron directory
wpa_supplicant: Successfully initialized wpa_supplicant
Failed to open config file '/system/sdcard/config/wpa_supplicant.conf', error: No such file or directory
Failed to read or parse configuration '/system/sdcard/config/wpa_supplicant.conf'.
udhcpc: udhcpc (v1.22.1) started
Sending discover...
Sending discover...
Sending discover...
No lease, forking to background
Initialized gpios
dropbear: 
bftpd: 
Created new certificate for webserver
lighttpd: 
Startup finished!

The SSID and the password is okay and i didn´t touch anythign more

Any suggestions?
thanks a lot in advance

**Comment by egidoval** [comment]
I forgot to add that the camera is not showing either on my router or by any other of my devices. It just doesn´t connect.
Thanks

**Comment by Dopeyr** [comment]
>Failed to open config file '/system/sdcard/config/wpa_supplicant.conf', error: No such file or directory

Did you rename `wpa_supplicant.conf.dist` to `wpa_supplicant.conf`?

**Comment by egidoval** [comment]
Yes i did

**Comment by Dopeyr** [comment]
Odd, can you double check? The error is saying that the file is not there.

**Comment by egidoval** [comment]
Yes, the file is there, if i take the sd card and i open it in the browser i can see there´s the file, and the filename is correct.

**Comment by Dopeyr** [comment]
Strange. I'm out of ideas on that one sorry.
